src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/declarativePod.groovy

```diff
@@ -0,0 +1,29 @@
+pipeline {
+  agent {
+    kubernetes {
+      label 'mypod'
+      podTemplate {
```

Comment:
This makes sense to me, but I'd like the label to be under podTemplate to reflect the actual structure of the PodTemplate.